doc/manual/manual.tex

```diff
@@ -8213,6 +8213,28 @@ \subsubsection{Melt migration in a 2D mid-ocean ridge model}
         effects might also play a role).
 \end{itemize}
 
+\subsubsection{Adiabatic boundary as initial temperature conditions}
+\textit{This section was contributed by Emmanuel Njinju, and D. Sarah Stamps}
+
+The adiabatic boundary is an initial temperature condition that allows for discretizing the model domain into layers separated by a user-defined isothermal boundary, which is an approximation of a thermal lithosphere definition. This geophysical set-up employs a table look-up approach. The user defines the adiabatic boundary, or lithospheric thickness in \aspect{} as an input ASCII data file that is formatted as 3 columns of `latitude', `longitude', and `depth', where `depth' is in kilometers. The first column must be in increasing latitude, then the corresponding second column in increasing longitude. The lithospheric geothermal gradient is linear with a default surface temperature of 273.15~K and a default temperature of 1673.15~K at the base of the lithosphere. Below the isotherm the temperature increases in an adiabatic manner at 0.5~K/km.
+
```

Comment:
`approximation of a thermal lithosphere definition` -> `approximation for the lithosphere's basal boundary`.

`as an input ASCII data file ` -> `through an ASCII data file`

`that is formatted as 3 columns of` -> `that contains columns for latitude, longitude and depth (km)`. Also, specify whether lat/lon are in degrees or radians.

`273.15~K ` -> `~ 273.15 K`, `1673.15~K ` -> `~ 1673.15 K`

`0.5~K/km` -> `~ 0.5 K/km`